Fix locations of STATEFILE and DEFAULT_MAIL_COMMAND for Debian.


Index: logrotate-3.8.0/config.h
===================================================================
--- logrotate-3.8.0.orig/config.h	2011-06-21 09:12:02.000000000 +0100
+++ logrotate-3.8.0/config.h	2011-08-28 18:37:14.646849933 +0100
@@ -29,7 +29,7 @@
  * Default settings for Linux - leave these last.
  */
 #ifndef DEFAULT_MAIL_COMMAND
-#define DEFAULT_MAIL_COMMAND "/bin/mail"
+#define DEFAULT_MAIL_COMMAND "/usr/bin/mail"
 #endif
 
 #ifndef COMPRESS_COMMAND
@@ -45,5 +45,5 @@
 #endif
 
 #ifndef STATEFILE
-#define STATEFILE "/var/lib/logrotate.status"
+#define STATEFILE "/var/lib/logrotate/status"
 #endif
